lib/matplotlib/backends/backend_ps.py

```diff
@@ -761,12 +761,10 @@ def _is_transparent(rgb_or_rgba):
 
 class GraphicsContextPS(GraphicsContextBase):
     def get_capstyle(self):
-        return {'butt': 0, 'round': 1, 'projecting': 2}[
-            GraphicsContextBase.get_capstyle(self)]
+        return {'butt': 0, 'round': 1, 'projecting': 2}[super().get_capstyle()]
```

Comment:
Is there a reason not to
```suggestion
        return {'butt': 0, 'round': 1, 'projecting': 2}[self.get_capstyle()]
```

Reply:
Because this is `get_capstyle`, and that would be recursive.

Reply:
Oh, :sheep: